contracts/external/cw-payroll/src/contract.rs

```diff
@@ -144,12 +283,14 @@ pub fn execute_receive(
     info: MessageInfo,
     wrapped: Cw20ReceiveMsg,
 ) -> Result<Response, ContractError> {
-    let config = CONFIG.load(deps.storage)?;
+    let sender = deps.api.addr_validate(&info.sender.clone().into_string())?;
```

Comment:
Are we 100% this sender will be the appropriate cw20 contract and not just anyone passing variables?